Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
+      event_sourcery (>= 0.14.0)
       pg
-      sequel (~> 4.38)
-    highline (1.7.8)
-    method_source (0.8.2)
-    mustermann (1.0.0)
+      sequel (>= 4.38)
+    highline (2.0.0)
+    method_source (0.9.2)
+    mustermann (1.0.3)
```

Comment:
### [mustermann](https://github.com/sinatra/mustermann)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.0.0 → 1.0.3

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **43** commits. See the full changes on [the compare page](https://github.com/sinatra/mustermann/compare/v1.0.0...v1.0.3).

These are the first 10 commits:
- (7c65d96) [Updating readme to list Ruby 2.2 as minimum](https://github.com/sinatra/mustermann/commit/7c65d9637ed81c194e3d05f0ccf3cfe76f0cf53e)
- (31fadc8) [Merge pull request #62 from cassidycodes/update-readme](https://github.com/sinatra/mustermann/commit/31fadc8c8ed3fd20482c69bd2b994f376315ae34)
- (119a61f) [Fix rendering of HTML table](https://github.com/sinatra/mustermann/commit/119a61f0e589cb9e917d8c901800a202bb66ff3b)
- (eb1a7ac) [Merge pull request #63 from stevenwilkin/patch-1](https://github.com/sinatra/mustermann/commit/eb1a7ac25625037987c204383d1e63bb9d53ee98)
- (3c88093) [Update .travis.yml](https://github.com/sinatra/mustermann/commit/3c880937ed44bff08ce2e52acb5ab41694522ad1)
- (0ba0411) [Merge pull request #65 from junaruga/feature/update-travis](https://github.com/sinatra/mustermann/commit/0ba04111db3018c8ebfed331131017ef7aa7169c)
- (d3c9531) [avoid unintended conflict of namespace, fixes #64](https://github.com/sinatra/mustermann/commit/d3c9531d372522d693fa5f768f13dbaa1d881d88)
- (04de221) [Update summary and description in gemspec file.](https://github.com/sinatra/mustermann/commit/04de221a809527c2be8c3f08c40a4fcd53f2bd53)
- (2245c8b) [Merge pull request #67 from junaruga/feature/update-text-in-…](https://github.com/sinatra/mustermann/commit/2245c8b31f2ebcb47abd2110c20fae6ddce666b2)
- (6f2a904) [Merge pull request #66 from sinatra/fix-64](https://github.com/sinatra/mustermann/commit/6f2a904acdb55c0b0a999cca4843f7bb40dbe961)


</details>